src/App.tsx

```diff
@@ -1,18 +1,37 @@
-import React from 'react';
+import React, { useMemo, useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination/Pagination';
 
+enum DefaultPageValues {
+  startPage = 1,
+  defaultPageSize = 5,
+}
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
It is worth writing in such a style that there is no need to disable the linter rules.